bin/create-exercise

```diff
@@ -22,8 +22,13 @@ while [[ ${pascal} =~ (.*)-(.*) ]]; do
     pascal=${BASH_REMATCH[1]}${BASH_REMATCH[2]^}
 done
 
+if [[ -z $author ]]; then
```

Comment:
I don't see how `$author` is initialized, unless it's an environment variable.

Reply:
read is a builtin command that sets environment variables.

```
read -rp "What's your GitHub handle? "
```
sets `REPLY`

```
read -rp "What's your GitHub handle? " author
```
sets `author`

https://www.gnu.org/software/bash/manual/html_node/Bash-Builtins.html#index-read

https://www.computerhope.com/unix/bash/read.htm

Reply:
Other tracks using this same environment variable:


```
8th/bin/add-exercise:    read -rp 'Your github username: ' author
arm64-assembly/bin/create-exercise:    read -rp "What's your github username? " author
idris/bin/create-exercise.sh:    read -rp "What's your github username? " author
jq/bin/add-exercise:read -rp 'Your github username: ' author
julia/bin/add-practice-exercise:    read -rp 'Your GitHub username: ' author
mips/bin/create-exercise:    read -rp "What's your github username? " author
r/bin/add-practice-exercise:    read -rp 'Your GitHub username: ' author
roc/bin/add-exercise:    read -rp 'Your github username: ' author
vlang/bin/bootstrap_practice_exercise.sh:    read -rp "What's your github username? " author
x86-64-assembly/bin/create-exercise:    read -rp "What's your github username? " author
```

Reply:
Believe me, I'm well aware how read works. I'm talking about line 25: `[[ -z $author ]]` -- when will the variable ever _not_ be empty at line 25? I'm questioning the purpose of this PR.

BTW read does not set _environment_ variables, it sets a _shell_ variable. It's up to `export` or `declare -x` to make it available in the environment. This is a pet peeve of mine.

Reply:
Sorry, I understand the question better now.

Before many tracks had `bin/create-exercise` scripts, I write my own for each track I was contributing to, that would copy in track-specific files and update .meta/config.json with my github handle.

Now that many tracks have scripts, I should use those. My concern is that I might mistype my github handle. So I set `author` in my .profile.

It isn't important, we don't need this PR.